frontend/src/components/CustomModal.js

```diff
@@ -1,6 +1,6 @@
-import React, { useState } from "react";
-import { Modal, Button } from "react-bootstrap";
-import "../styles/CustomModal.css";
+import React, { useState } from 'react';
+import { Modal, Button } from 'react-bootstrap';
+import '../styles/CustomModal.css';
```

Comment:
We should not see these type of changes in diff, we need to agree on shared formatting configuration - let's get back to this next sprint.